src/graph/service/GraphServer.cpp

```diff
@@ -24,7 +24,7 @@ bool GraphServer::start() {
   int numThreads =
       FLAGS_num_worker_threads > 0 ? FLAGS_num_worker_threads : gServer_->getNumIOWorkerThreads();
   std::shared_ptr<apache::thrift::concurrency::ThreadManager> threadManager(
-      PriorityThreadManager::newPriorityThreadManager(numThreads, false /*stats*/));
```

Comment:
Why change this?

Reply:
After updating third party 3.0，newPriorityThreadManager remove second argument

Reply:
This argument affects the performance a lot, we must know how it behaves now. And make sure if it has another way to set the value. A benchmark must done for this change.

Reply:
https://github.com/facebook/fbthrift/commit/bfb527c7b254230cf5ea6ceb818c8ce57a3f9a84?diff=split
Through this commit，thrift removed stats, and the function is the same as setting it to false